File: Java/1074 - Even or Odd.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cases = sc.nextInt(), number;
        for (int count = 0; count < cases; count++) {
            number = sc.nextInt();
            if (number == 0)
                System.out.println("NULL");
            else if (number%2!=0) {
                if (number > 0)
                    System.out.println("ODD POSITIVE");
                else    
                    System.out.println("ODD NEGATIVE");
            }
            else {
                if (number > 0)
                    System.out.println("EVEN POSITIVE");
                else    
                    System.out.println("EVEN NEGATIVE");
            }
        } 
    
    }
}

File: Python/1074 - Even or Odd.py
n = int(input())
for i in range(n):
    x = int(input())
    if x < 0:
        if x % 2 == 1:
            print("ODD NEGATIVE")
        elif x % 2 == 0:
            print("EVEN NEGATIVE")
    elif x > 0:
        if x % 2 == 1:
            print("ODD POSITIVE")
        elif x % 2 == 0:
            print("EVEN POSITIVE")
    else:
        print("NULL")